lib/linguist/generated.rb

```diff
@@ -82,7 +82,8 @@ def generated?
       generated_roxygen2? ||
       generated_jison? ||
       generated_yarn_lock? ||
-      generated_grpc_cpp?
+      generated_grpc_cpp? ||
+      generated_jest_snap?
```

Comment:
Could you add it after `generated_by_zephir?` in the list? We prefer to place first the rules that don't require opening the file (to open it only if necessary).